app/readme.md

```diff
@@ -51,7 +46,7 @@ An alternative on **macOS** is to install the tools locally with `brew install g
 If you are on **Windows** (without WSL2), run the following command:
 
 ```sh
-docker run --pull always --rm -w /app -v %cd%:/app registry.gitlab.com/couchers/grpc sh -c "cat generate_protos.sh | dos2unix | sh"
+docker run --pull always --rm -w /app -v ${PWD}/app:/app registry.gitlab.com/couchers/grpc sh -c "apt-get update && apt-get install dos2unix && cat generate_protos.sh | dos2unix | sh"
```

Comment:
solved command on windows

Reply:
If `dos2unix` is not in the container, then I think we should add it there instead of installing it here

Reply:
sure, thing is, I don't have access to it 🙃

Reply:
It's here: https://github.com/Couchers-org/grpc

Reply:
I've installed it there now, so no need to install here!

Reply:
okey, thank you!!